### ISSUE TITLE
GUI: Request for comments

### ISSUE THREAD

**Issue by FROGGS** [kthakore/Zombie-Trap#11]
<img src="http://i.imgur.com/SxvRm.png">

**Comment by kthakore** [comment]
OK I like the buttons for this so far. Can you explain the numbers are? Also we need a row to assign classes of zombies. Like you click on Exploding Button and Click on Zombie. He will turn into an exploding type. Either you can make it on the bottom or make a GUI arrow [>] that when you click. I would prefer if you had the character assign buttons at the top.

http://i.imgur.com/rEs9P.png

**Comment by kthakore** [comment]
Like this: ![Comments on Image](http://i.imgur.com/rEs9P.png)

**Comment by FROGGS** [comment]
These two buttons at lower left are the zombie classes. 
You click on such a button (first one is normal zombie), then choose a zombie by clicking on it. 
The number tells you how many zombies you can change that way.

The second button should be the "converting zombie" who can transform humans. But I guess there should be a better image.

I thought its better if the zombie class buttons are at the bottom because the zombies are walking at the "bottom"... no?

**Comment by kthakore** [comment]
Yeah you are right. This is fine sorry I got confused. We need blowing up and digging/beserker.

**Comment by FROGGS** [comment]
Ya, cool!
What do you think about that:
Normal zombies hate sunlight and die if they are to much in lighter reagons. This way we can make a zombie class that can stand the sunlight longer...

Maybe i can make a pic to show what i mean.

kthakore reply@reply.github.com hat geschrieben:

**Comment by kthakore** [comment]
Sounds like an awesome idea to add to a level.

**Comment by FROGGS** [comment]
Okay, so we have:
- normal zombie
- converter guy who can transform human to zombies: how many?
- an exploding zombie: when does it explode? instead of flipping direction? on comand?
- a digging zombie: pphammer style! (but maybe with its hands), one type for digging through walls, one for digging through floor
- a climber
- a stopper: like a solid wall for other zombies (exploding ones will explode for example)
- a zombie who can stand sunlight better

As long as a zombie "lives" it can be converted to another class, again and again...
We need the goal (something like a door, or a vault). Or they should spawn from a vault... whatever fits better to the theme.

What do you think?

**Comment by kthakore** [comment]
I like this!. 
On Tue, 2011-08-16 at 12:35 -0700, FROGGS wrote:

> Okay, so we have:
> - normal zombie
> - converter guy who can transform human to zombies: how many?

Converter can keep converting indefinitely. Until he is killed. 

> - an exploding zombie: when does it explode? instead of flipping direction? on comand?

Timing I think. 

> - a digging zombie: pphammer style! (but maybe with its hands), one type for digging through walls, one for digging through floor

pphamer?

> - a climber

yup!

> - a stopper: like a solid wall for other zombies (exploding ones will explode for example)
>   yup!
> - a zombie who can stand sunlight better

absolutely. He should spark in sunlight and can stand it for only a
longer amount of time. Then he dies. 

> As long as a zombie "lives" it can be converted to another class, again and again...
> We need the goal (something like a door, or a vault).

The town! 

> Or they should spawn from a vault... whatever fits better to the theme.
> 
> From Dr. Frank's Mobile Laboratory! I drives in ever starting of the
> level, various places. The town can be located other places too. 
> 
> What do you think?

Should we make the user bring zombies back to lab to save them for next
round? 

## 

Kartik Thakore thakore.kartik@gmail.com

**Comment by FROGGS** [comment]
> pphamer?

http://en.wikipedia.org/wiki/P._P._Hammer_and_his_Pneumatic_Weapon

> Should we make the user bring zombies back to lab to save them for next round?

Hmmm, I guess that would be too hard. The amount of zombies per level shouldnt alter (unless we make a unicorn/normal/hardcore -difficulty-mode).

**Comment by kthakore** [comment]
ok, maybe we do varying hardness later

Kartik Thakore
